stock_picking_analytic/views/stock_picking.xml

```diff
@@ -8,31 +8,46 @@
         <field name="model">stock.picking</field>
         <field name="inherit_id" ref="stock.view_picking_form" />
         <field name="arch" type="xml">
-            <field name="picking_type_id" position="after">
+            <xpath expr="//group/group/field[@name='picking_type_id']" position="after">
                 <field
                     name="analytic_account_id"
                     groups="analytic.group_analytic_accounting"
                     options="{'no_create': True}"
                 />
-            </field>
-            <field name="move_ids_without_package" position="attributes">
-                <attribute
-                    name="context"
-                    operation="python_dict"
-                    key="default_analytic_account_id"
-                >
-                    analytic_account_id
+            </xpath>
+            <xpath
+                expr="//notebook/page[@name='operations']
+                /field[@name='move_ids_without_package']"
+                position="attributes"
+            >
+                <attribute name="context">
+                    {'default_company_id': company_id,
+                    'default_date': scheduled_date,
+                    'default_date_deadline': date_deadline,
+                    'picking_type_code': picking_type_code,
+                    'default_picking_id': id,
+                    'form_view_ref':'stock.view_move_form',
+                    'address_in_id': partner_id,
+                    'default_picking_type_id': picking_type_id,
+                    'default_location_id': location_id,
+                    'default_location_dest_id': location_dest_id,
+                    'default_partner_id': partner_id,
+                    'default_analytic_account_id': analytic_account_id}
                 </attribute>
-            </field>
-            <field name="move_line_ids_without_package" position="attributes">
-                <attribute
-                    name="context"
-                    operation="python_dict"
-                    key="default_analytic_account_id"
-                >
-                    analytic_account_id
+            </xpath>
+            <xpath
+                expr="//notebook/page[@name='detailed_operations']
+                /field[@name='move_line_ids_without_package']"
+                position="attributes"
+            >
+                <attribute name="context">
+                    {'tree_view_ref': 'stock.view_stock_move_line_detailed_operation_tree',
+                    'default_picking_id': id, 'default_location_id': location_id,
+                    'default_location_dest_id': location_dest_id,
+                    'default_company_id': company_id,
+                    'default_analytic_account_id': analytic_account_id}
                 </attribute>
```

Comment:
Why didn't you leave this as it was before? This module inherits from `base_view_inheritance_extension` which is supposed to allow us not having to rewrite all the context keys. Didn't it work unchanged?